static_src/test/functional/user_role.spec.js

```diff
@@ -1,58 +1,150 @@
 
-describe('User roles', function () {
-  let cookieResult;
-
-  beforeEach(function () {
-    browser.url('/');
-  });
+import UserRoleElement from './pageobjects/user_role.element';
 
-  afterEach(function () {
-    browser.deleteCookie('testing_user_role');
-  });
+describe('User roles', function () {
+  let cookieResult,
+    userRoleElement,
+    pageUrl,
+    cookieValue;
+  const guidManagerOrgX = 'org-manager-x-uid-601d-48c4-9705',
+    guidManagerOrgY = 'org-manager-y-uid-601d-48c4-9705',
+    cookieManagerOrgY = 'org_manager_org_y',
+    cookieManagerOrgX = 'org_manager_org_x',
+    cookieManagerSpaceXX = 'space_manager_space_xx',
+    cookieManagerSpaceYY = 'space_manager_space_yy',
+    urlOrgY = '/#/org/user_role-org_y-ffe7-4aa8-8e85-94768d6bd250',
+    urlOrgX = '/#/org/user_role-org_x-ffe7-4aa8-8e85-94768d6bd250';
 
   describe('User role cookie test', function () {
```

Comment:
is this section needed since all the setCookie and getCookie functionality is now in the userRoleElement?